_docs-v3/toolbar/titleRangeSeparator.md

```diff
@@ -0,0 +1,34 @@
+---
+title: titleRangeSeparator
+---
+
+The date range in the **agendaWeek** title is, by default, separated by " - " between the beginning and end date.
```

Comment:
for any string likes `" - "` and `" to "`, in this sentence and in other sentences in this article, please remember to put backticks around it so that it looks  like a small inline code snipped